src/boot/efi/linux_x86.c

```diff
@@ -19,17 +19,20 @@
 #include "util.h"
 
 #define KERNEL_SECTOR_SIZE 512u
-#define SETUP_MAGIC             0x53726448      /* "HdrS" */
+#define BOOT_FLAG_MAGIC    0xAA55u
+#define SETUP_MAGIC        0x53726448u /* "HdrS" */
```

Comment:
given these are filled into structs that use proper uint32_t, uint16_t and so on, maybe use UINT32_C() and UINT16_C() here?